packages/client/hmi-client/src/model-representation/petrinet/nested-petrinet-renderer.ts

```diff
@@ -235,7 +235,7 @@ export class NestedPetrinetRenderer extends PetrinetRenderer {
 		transitions
 			.append('text')
 			.attr('y', (d) => -d.height / 2 - 8)
-			.style('font-family', 'STIX Two Text, serif')
```

Comment:
Could we make sure this font is not used anywhere else and remove it if that is the case.

Reply:
`STIX Two Text` is completely gone.

Do you want that serif removed?
`serif` is used by itself in a `Tera-Model-Template` as a list style
In `Tera-Model-Comparation`  legend-circle & square class as `'Times New Roman', Times, serif`